src/App.tsx

```diff
@@ -17,11 +17,11 @@ import {
   browserLocalPersistence, 
   onAuthStateChanged
 } from 'firebase/auth';
-import apolloClient from './api/apolloClient';
-import { ApolloProvider } from '@apollo/client';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { ReactQueryDevtools } from 'react-query/devtools';
 
+export const endpoint = 'http://localhost:9000/graphql';
+
```

Comment:
Will be changed to the deployed backend, should look into authentication on the backend to only allow verified users to call the backend